modules/swagger-core/src/test/scala/1_3/converter/SnakeCaseConverterTest.scala

```diff
@@ -95,7 +95,9 @@ class SnakeCaseConverter extends ModelConverter {
         val name = (model.asInstanceOf[ModelImpl]).getName
         if (model.isInstanceOf[ModelImpl]) {
```

Comment:
Here we have another issue that doesn't relate to the initial one. The [ComposedModel](https://github.com/swagger-api/swagger-core/blob/400662d7253c0c2efb14316f8948ddc61a60069e/modules/swagger-models/src/main/java/io/swagger/models/ComposedModel.java) is not supported here. So, this converter will update the references to composed models but leave model names untouched.